WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/EditorPage.java

```diff
@@ -5,52 +5,100 @@
 
 import org.wordpress.android.R;
 
+import static android.support.test.InstrumentationRegistry.getInstrumentation;
 import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
+import static android.support.test.espresso.Espresso.pressBack;
 import static android.support.test.espresso.action.ViewActions.click;
 import static android.support.test.espresso.action.ViewActions.typeText;
 import static android.support.test.espresso.assertion.ViewAssertions.matches;
 import static android.support.test.espresso.matcher.RootMatchers.isDialog;
-import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
```

Comment:
I'm getting a compiler error because this was removed but `isDescendantOfA` is still being used for the `publishConfirmation` below. Adding this line back resolves that error.

Reply:
🤦‍♂ I used GitHub suggestion to pull in a change from James, overlooked the import, my bad!